build/charts/theia/README.md

```diff
@@ -49,10 +49,11 @@ Kubernetes: `>= 1.16.0-0`
 | clickhouse.storage.size | string | `"8Gi"` | ClickHouse storage size. Can be a plain integer or as a fixed-point number using one of these quantity suffixes: E, P, T, G, M, K. Or the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki. |
 | clickhouse.ttl | string | `"12 HOUR"` | Time to live for data in the ClickHouse. Can be a plain integer using one of these unit suffixes SECOND, MINUTE, HOUR, DAY, WEEK, MONTH, QUARTER, YEAR. |
 | grafana.dashboards | list | `["homepage.json","flow_records_dashboard.json","pod_to_pod_dashboard.json","pod_to_service_dashboard.json","pod_to_external_dashboard.json","node_to_node_dashboard.json","networkpolicy_dashboard.json","network_topology_dashboard.json"]` | The dashboards to be displayed in Grafana UI. The files must be put under provisioning/dashboards. |
+| grafana.default_theme | string | `"light"` | Default grafana visual theme |
```

Comment:
could you expand a little bit more on why we need to change the default theme? Is it a must?

Reply:
Yeah, it's just because the muted colors used by the plugin don't match the vibrancy or theme of the colors in the other dashboards/plugins.